src/EnergyPlus/TARCOGOutput.cc

```diff
@@ -1312,19 +1309,19 @@ namespace TARCOGOutput {
 			gio::write( WINCogFile, Format_2020 ) << xwght( i );
 			gio::write( WINCogFile, Format_2031 );
 			gio::write( Format_2020 ) << xgcon( 1, 1 );
-			for ( j = 2; i <= 3; ++j ) {
+			for ( j = 2; j <= 3; ++j ) {
```

Comment:
Was this just a straight-up bug fix?

Reply:
Yup, this is bug fix done in WINDOW however, here I noticed it because compiler gave me warning messages.